Lecture-2/Task3/task3/Application.java

```diff
@@ -7,21 +7,23 @@
     private static String operand = "";
     private static boolean isAppQuit = false;
 
-    private static void getResult(String operand, double firstDigit, double secondDigit) {
-
+    public static double getResult(String operand, double firstDigit, double secondDigit) {
```

Comment:
`firstDigit` -> `firstOperand` - since it may be not a digit, but number
`secondDigit` -> `secondOperand`